### DISCUSSION TITLE
After clicking [Save], jump to the [Show] page

### DISCUSSION THREAD

**Discussion by yt-dev** [refinedev/refine#4688] [Q&A] [answered]
Refine version: 4.x
The components generated by Refine feature Inferencer. Just wanna go back to show page after updating content on Edit page.

**Comment by BatuhanW** [comment]
Hello @yt-dev could you elaborate more on your problem?

**Reply by yt-dev** [reply, marked as answer]
Oops, forgot to update this question. It’s been solved. Ref: https://refine.dev/docs/packages/documentation/react-hook-form/useForm/#redirect.
Anyway, thanks for your reply :)
